readthedocsext/theme/templates/projects/search_analytics.html

```diff
@@ -55,3 +53,8 @@ <h3 class="ui small header">
     {% endif %}
   </div>
 {% endblock %}
+
+{% block project_edit_sidebar_help_topics %}
+  {% include "includes/elements/link.html" with url="https://docs.readthedocs.io/page/guides/search-analytics.html" text="How to use search analytics" is_external=True class="item" %}
+  {% include "includes/elements/link.html" with url="https://docs.readthedocs.io/page/guides/technical-docs-seo-guide.html" text="How to do Search Engine Optimization (SEO) for documentation projects" is_external=True class="item" %}
```

Comment:
Spelled out, the acronym is not a proper noun so there is no need to capitalize.

```suggestion
  {% include "includes/elements/link.html" with url="https://docs.readthedocs.io/page/guides/technical-docs-seo-guide.html" text="How to do search engine optimization (SEO) for documentation projects" is_external=True class="item" %}
```

Reply:
Oh strange, it's actually not written like this in the original page. Nice spot!